.ci_support/linux_64_.yaml

```diff
@@ -5,7 +5,7 @@ c_compiler_version:
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge,defaults
+- conda-forge
 channel_targets:
```

Comment:
same here ... channel_targets are for sure not conda-forge

Reply:
This is a merge from conda-forge and we don't use `.ci_support` so it should be an acceptable change. We've done the merge to make future merges easier.